frontend/src/metabase/public/components/EmbedFrame/EmbedFrame.module.css

```diff
@@ -25,7 +25,12 @@ body {
 }
 
 [data-metabase-theme="night"][data-metabase-theme="night"],
-[role="tooltip"] {
+/**
+  * role="tooltip" works with both legacy and Mantine popover,
+  * the legacy Popover however, has the role "tooltip", but with data-theme="popover",
+  * so we need to target exactly just tooltips.
+  */
+[role="tooltip"]:is([data-theme~="tooltip"], :global(.emotion-Tooltip-tooltip)) {
```

Comment:
I figure I should add some details for folks who're not familiar why this is needed, we in the past, theme dark theme everywhere using `isNightMode`. But with [[Epic] Re-design transparent theme for static embedding and public links](https://github.com/metabase/metabase/issues/43838#top), we've standardized theming and made it more sane.

One of the changes involves the removal of in-component style logics that decide when we need to apply white text or dark text (dark text on light mode, and white text on dark mode OR in the tooltip).